packages/server/src/migrations/schema/v87.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * This is a generated file
+ * Do not edit manually.
+ */
+
+import { PoolClient } from 'pg';
+
+export async function run(client: PoolClient): Promise<void> {
+  await client.query('ALTER TABLE IF EXISTS "Coding" ADD COLUMN IF NOT EXISTS "isSynonym" BOOLEAN');
+  await client.query('DROP INDEX CONCURRENTLY "Coding_display_idx"');
+  await client.query('ALTER INDEX IF EXISTS "Coding_display_trgm_idx" RENAME TO "Coding_display_idx"');
```

Comment:
This migration was tested against a `Coding` table containing a full import of UMLS on my laptop, and ran in 10 ms